src/js/load-package.ts

```diff
@@ -17,28 +17,28 @@ import { makeWarnOnce } from "./util";
 
 /**
  * Initialize the packages index. This is called as early as possible in
- * loadPyodide so that fetching repodata.json can occur in parallel with other
+ * loadPyodide so that fetching pyodide-lock.json can occur in parallel with other
  * operations.
  * @param lockFileURL
  * @private
  */
 async function initializePackageIndex(lockFileURL: string) {
-  let repodata;
+  let lockfile;
   if (IN_NODE) {
     await initNodeModules();
     const package_string = await nodeFsPromisesMod.readFile(lockFileURL);
-    repodata = JSON.parse(package_string);
+    lockfile = JSON.parse(package_string);
   } else {
     let response = await fetch(lockFileURL);
-    repodata = await response.json();
+    lockfile = await response.json();
   }
-  if (!repodata.packages) {
+  if (!lockfile.packages) {
     throw new Error(
-      "Loaded repodata.json does not contain the expected key 'packages'.",
+      "Loaded pyodide lock file does not contain the expected key 'packages'.",
     );
   }
-  API.repodata_info = repodata.info;
-  API.repodata_packages = repodata.packages;
+  API.repodata_info = lockfile.info;
+  API.repodata_packages = lockfile.packages;
```

Comment:
Probably we should update the variable name in `micropip` and here together.

Reply:
For compatibility reasons we could store it into both `repodata_xxx` and `lockfile_xxx` for a while.